pkg/normalize/validate.go

```diff
@@ -696,6 +697,7 @@ var providerCapabilityChecks = []pairTypeCapability{
 	capabilityCheck("SRV", providers.CanUseSRV),
 	capabilityCheck("SSHFP", providers.CanUseSSHFP),
 	capabilityCheck("TLSA", providers.CanUseTLSA),
+	capabilityCheck("DHCID", providers.CanUseDHCID),
```

Comment:
Please keep this list alphabetized. (and if anything else is out of place, also move it)